squad-server/plugins/db-log.js

```diff
@@ -43,6 +43,7 @@ export default class DBLog extends BasePlugin {
     super(server, options, connectors);
 
     this.models = {};
+    this.playerSession = {};
```

Comment:
I don't think there's a need for this object or any of the logic related to it?

Reply:
After testing without the object declaration above it would throw an error. So I believe it is needed @Thomas-Smyth